nxt_editor/stage_view.py

```diff
@@ -863,11 +881,14 @@ def mouseReleaseEvent(self, event):
                 if type(items_released_on[1]) is NodeGraphicsPlug:
                     dropped_plug = items_released_on[1]
                     dropped_node_path = dropped_plug.parentItem().node_path
+                    locked = dropped_plug.parentItem().locked
```

Comment:
Since we have the node path of the dropped node, can we look up locked instead of trusting the graphic here?